### PR TITLE
[OneLineFix/TCPreplay] Replaced file_cache by preload_pcap

### PR DESCRIPTION
**Pull request by gpotter2**
Fixes https://github.com/secdev/scapy/issues/488

### CODE REVIEW

File: scapy/sendrecv.py [hunk 1]
```diff
@@ -286,13 +286,13 @@ def sendp(x, inter=0, loop=0, iface=None, iface_hint=None, count=None, verbose=N
                       verbose=verbose, realtime=realtime, return_packets=return_packets)
 
 @conf.commands.register
-def sendpfast(x, pps=None, mbps=None, realtime=None, loop=0, file_cache=False, iface=None):
+def sendpfast(x, pps=None, mbps=None, realtime=None, loop=0, preload_pcap=False, iface=None):
```
Comment (p-l-) [inline scapy/sendrecv.py]: Please keep the same API. This is not supposed to be a `tcpreplay` option, but a Scapy function parameter.

File: scapy/sendrecv.py [hunk 2]
```diff
@@ -286,13 +286,13 @@ def sendp(x, inter=0, loop=0, iface=None, iface_hint=None, count=None, verbose=N
...
     """Send packets at layer 2 using tcpreplay for performance
     pps:  packets per second
     mpbs: MBits per second
     realtime: use packet's timestamp, bending time with real-time value
     loop: number of times to process the packet list
-    file_cache: cache packets in RAM instead of reading from disk at each iteration
+    preload_pcap: preloads packets into RAM before sending
```
Comment (p-l-) [inline scapy/sendrecv.py]: Same here :)

**Comment by gpotter2** [comment]
Done

**Comment by gpotter2** [comment]
@p-l- Please restart the failing build :)

**Comment by p-l-** [comment]
I have canceled the MacOS builds and merged anyway since Travis-CI [is experiencing serious MacOS infrastructure issues](https://www.traviscistatus.com/incidents/k79mjcv403c4).

**Pull request merged**